files/test_config.py

```diff
@@ -22,9 +22,10 @@
 
 Crawler Configuration
 '''
-cwd = os.getcwd()
+root = os.environ['TEST_DATA']
```

Comment:
I don't see the use of the environment variable before and the code here assumes that it is always present

Reply:
It is set in `run_tests.sh`. See the commit message for explanation.